File: GLExample/app/src/main/java/com/blueberry/glexampe/MainActivity.java
package com.blueberry.glexampe;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.opengl.GLSurfaceView;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {
    static {
        System.loadLibrary("glejni");
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.btnGl2SurfaceView).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
               Intent  intent = new Intent(MainActivity.this,GL2SurfaceViewActivity.class) ;
               MainActivity.this.startActivity(intent);
            }
        });

        findViewById(R.id.btnGlSurfaceView).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent  intent = new Intent(MainActivity.this,SurfaceViewActivity.class) ;
                MainActivity.this.startActivity(intent);
            }
        });

        findViewById(R.id.btnEGLSurfaceView).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent  intent = new Intent(MainActivity.this,EGLSurfaceActivity.class) ;
                MainActivity.this.startActivity(intent);
            }
        });
    }

}

File: audioeffect/app/src/main/java/com/blueberry/audioeffect/MainActivity.kt
package com.blueberry.audioeffect

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import kotlin.concurrent.thread

class MainActivity : AppCompatActivity() {
    companion object {
        private const val TAG = "MainActivity"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<View>(R.id.btn_equalizer).setOnClickListener {
            val intent = Intent(this@MainActivity, EqualizerActivity::class.java)
            this@MainActivity.startActivity(intent)
        }
        findViewById<View>(R.id.btn_compressor).setOnClickListener { }
        findViewById<View>(R.id.btn_reverb).setOnClickListener { }

        Log.i(TAG, "onCreate: ")
        thread {
            Log.i(TAG, "onCreate: thread block")
            ResourceUtil.writeIfAbsent(this)
        }
    }
}